app/assets/javascripts/backbone/views/loan_view.coffee

```diff
@@ -0,0 +1,21 @@
+class MS.Views.LoanView extends Backbone.View
+
+  el: '.loan'
+
+  events: (params) ->
+    'change #loan_primary_agent_id': 'removeFromSecAgentList'
+    'change #loan_secondary_agent_id': 'removeFromPryAgentList'
+
+  removeFromSecAgentList: ->
+    e = document.getElementById('loan_primary_agent_id')
+
+    selected = e.options[e.selectedIndex].value
+
+    $('#loan_secondary_agent_id').find('option[value=' + selected + ']').remove()
+
+  removeFromPryAgentList: ->
+    e = document.getElementById('loan_secondary_agent_id')
+
+    selected = e.options[e.selectedIndex].value
+
+    $('#loan_primary_agent_id').find('option[value=' + selected + ']').remove()
```

Comment:
This is a good first attempt but there is a problem here. If you accidentally select "Sally" from the primary agent list but she's really a secondary agent, you can never fix your mistake and select her from the secondary list. You'd have to refresh the page and start again.

While I appreciate the effort here, I would suggest removing this front-end functionality. I think the backend validation will be enough to keep data clean and provide a decent user experience. It's up to you @emitche and @linesarefuzzy as planners, but I think to get this working in JS is going to take quite a bit more work.

Reply:
Thanks so much for all of this work, @arisah-ijeoma.

Let's remove the front end portion of this change and keep the back-end portions. Thanks again.